core/src/main/resources/META-INF/fabric8/kind-fileindicator-mapping-default.adoc

```diff
@@ -0,0 +1,102 @@
+|===
```

Comment:
We should add a documentation in the header (as mentioned in a previous comment) and we should add that the canonical name should be the last part of the mapping (which I would say is the full kind in all lowercase).